src/Editor/Util/AssetConverterSkeleton.hpp

```diff
@@ -0,0 +1,51 @@
+#include <string>
+#include <assimp/Importer.hpp>
+#include <assimp/scene.h>
+#include <assimp/anim.h>
+#include <Engine/Animation/AnimationClip.hpp>
+
+class AssetConverterSkeleton {
+    public:
+        /// Constructor.
+        AssetConverterSkeleton();
+
+        /// Destructor.
+        ~AssetConverterSkeleton();
+
+        /// Convert a mesh from fbx to .asset format.
+        /**
+         * @param filepath Filepath of the fbxfile.
+         * @param destination Filepath of the destination inculding name and extension.
+         * @param triangulate Should the mesh be triangulated?
```

Comment:
triangulate, importNormals and importTangents are not a part of this function, but bindpose is.